3-platform/warehouse/spec/src/main/java/com/_6core/platform/warehouse/rest/v1/inventory/Inventory.java

```diff
@@ -0,0 +1,10 @@
+package com._6core.platform.warehouse.rest.v1.inventory;
+
+public record Inventory(
+        String inventoryId,
+        String warehouseId,
```

Comment:
we dont need to return warehouseId, mercant should know where is he store items. 
```suggestion
```

Reply:
Ok, thanks.